dagmc_stats.py

```diff
@@ -51,3 +51,4 @@ def get_entity_ranges(my_core, meshset, entity_types):
 
     return entity_ranges
 
+get_dagmc_tags(core.Core())
```

Comment:
Not sure why this line is here in the `dagmc_stats` module.  This line will be run every time, and `core.Core()` will be an empty instance of MOAB with no tags

Reply:
This isn't supposed to be here, it got deleted somewhere later.